notebooks/ton-year.ipynb

```diff
@@ -123,194 +89,79 @@
-    "        + \" ton-years\"\n",
-    "    )\n",
-    "    print(\n",
-    "        \"Benefit from 1tCO2 with delay: \"\n",
-    "        + str(round(method_output[\"benefit\"], 2))\n",
-    "        + \" ton-years\"\n",
-    "    )\n",
-    "    print(\n",
-    "        \"Number needed: \" + str(round(method_output[\"num_for_equivalence\"], 1))\n",
-    "    )\n",
-    "    print()\n",
-    "    if plot:\n",
-    "        plot = plt.plot(method_output[\"baseline\"], label=\"baseline\")\n",
-    "        plot = plt.plot(method_output[\"scenario\"], label=\"scenario\")\n",
-    "        plot = plt.legend()"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "f80db065",
-   "metadata": {},
-   "source": [
-    "And a function implementing the three ton-year accounting set-ups:\n"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "47acc1f6",
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "def calculate_tonyears(method, baseline, time_horizon, delay, discount_rate):\n",
-    "\n",
-    "    if delay < 0:\n",
-    "        raise ValueError(\"Delay cannot be negative.\")\n",
-    "    if time_horizon <= 0:\n",
-    "        raise ValueError(\"Time horizon must be greater than zero.\")\n",
-    "    if len(baseline) < time_horizon:\n",
-    "        raise ValueError(\n",
-    "            \"Method cannot analyze over a time horizon that is longer than the baseline scenario (i.e. the length of the baseline array).\"\n",
-    "        )\n",
-    "    if method not in [\"mc\", \"ipcc\", \"lashof\"]:\n",
-    "        raise ValueError(\n",
-    "            \"No ton-year accounting method called '\"\n",
-    "            + method\n",
-    "            + \"'. Options include: Moura-Costa ('mc'), Lashof ('lashof'), and IPCC ('ipcc').\"\n",
-    "        )\n",
-    "\n",
-    "    # All methods calculate the baseline cost of emitting 1tCO2 at t=0 as the\n",
-    "    # atmospheric ton-years incurred over the period 0<=t<=time_horizon.\n",
-    "    time_horizon_timesteps = time_horizon + 1\n",
-    "    baseline = baseline[:time_horizon_timesteps]\n",
-    "    baseline_discounted = get_discounted_curve(discount_rate, baseline)\n",
-    "    baseline_atm_cost = np.trapz(baseline_discounted)\n",
-    "\n",
-    "    if method == \"mc\":\n",
-    "        # The Moura-Costa method calculates the ton-year benefit of a delayed emission\n",
-    "        # as the ton-years of carbon storage outside of the atmosphere over the period\n",
-    "        # 0<=t<=delay. Moura-Costa ignores the atmospheric impact of post-storage re-emission.\n",
-    "        delay_timesteps = delay + 1\n",
-    "        scenario = [-1] * (delay_timesteps) + [0] * (\n",
-    "            len(baseline) - delay_timesteps\n",
-    "        )\n",
-    "        scenario = get_discounted_curve(discount_rate, scenario)\n",
-    "        benefit = -np.trapz(scenario[:delay_timesteps])\n",
+    "### Parameters\n",
```

Comment:
Would be nice to point out that we have an 'overview' in the docs where we discuss the parameters (actually we even discuss the IRF curves too!) in more depth.